docs/scope.md

```diff
@@ -0,0 +1,113 @@
+
+#### HA Checker: Scenario SAP HANA Performance Optimized on Azure
+> FIXME: How do I releate this "HA Checker: Scenario SAP HANA Performance 
+> Optimized on Azure"
+This checker uses Azure's recommendations to provide an overview of how well
+the deployed SAP HANA cluster meets their indicators. More details about this
+recommendation can be seen [here](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker)
+
+#### HA Checker: Scenario Pacemaker on Azure
+This check is also comparing against Azure's recommendations for a Pacemaker
+cluster.
+> FIXME: Link to this one?
+
+#### More coming
+We are working on implementing more checkers also for other cloud providers.
+
+### Discovery
+> FIXME: Based on what I wrote in [Checks](#checks) I think we should probably
+> have a separate part here: Discovery are the parts of trento which are
+> attempting to detect e.g. the role of a SAP HANA node and the processes that
+> run on each node; while the checkers are additionally checking those parameters
+> against predefined values.
+
+## Clusters
+This view currently allows to check the status of the all the discovered HANA DB
+clusters, including the information of the status of each node and the role
+associated to each node. This view also allows to see the resources, their type
+and distribution within the cluster nodes.
+
+## Environments
+The environments view currently only shows a detail of the landscapes that
+are part of an environment.
+
+## Landscapes
+This view at the moment shows a list of the discovered landscapes, detailing the
+count of SAP systems inside each of them as well as the total number of hosts
+that belong to these SAP systems.
+
+## SAP Systems
+In this view are listed the SAP systems, usually identified by a SID or 
+`SAP System Idenfication` string such as `PRD`, `DEV`, or `QAS`. These are used
+often to identify productive, development and quality assurance systems.
+In this view we get an overview of the distribution of each SID, the number of
+hosts disvered of each of them and additional details.
+
+## Agents
+Though not specifically a view/section, as it's the core component of `trento`,
+it's important to understand what they are.
+
+The agents are the `trento` processes that run in each of the nodes that conform
+a highly available SAP Applications cluster. On each of these, the agents attempt
+to discover the role of the node and run a set of checks which result in the
+recommendations described above in the [Checkers](#checkers).
+The information that is gathered by them is stored through a distributed KV store
+and is made visible on the `trento` web UI through the `trento` web UI component.
+
+The agents implement the core functionality of trento. They are responsible for
+the discovery of all the clustered components that are required in order to run
+highly available SAP Applications. These agents implement discovery for:
+  - Pacemaker
+  - Corosync
```

Comment:
Corosync data gathering is not implemented yet, but well, we can put it here hehe